client/src/components/Q&A/AddQModal.jsx

```diff
@@ -23,10 +23,30 @@ export default function AddQModal({ show, setShow }) {
       </div>
       <div className="modal-content">
         <div className="modal-body">
-          <h4>Modal</h4>
```

Comment:
Is there a reason for doing a div instead of using the built in form tags that html uses?

Reply:
Doh! Made the switch to a form